spec/amp-story-player.md

```diff
@@ -126,3 +130,192 @@ CSS variable with the URL pointing to the poster image of the story.
   </a>
 </amp-story-player>
 ```
+
+## Programmatic Control
+
+Call the player's various methods to programmatically control the player. These methods are exposed on the DOM HTML element, `const playerEl = document.querySelector('amp-story-player')` and on the global class variable,`const player = new AmpStoryPlayer(window, playerEl)`.
+
+### Methods
+
+#### load
+
+Will initialize the player manually. This can be useful when the player is dynamically added such as when using frameworks like React.
+
+```
+function StoryPlayer({ url, title, poster, width = 360, height = 600 }) {
+
+       useEffect(() => {
+           const playerEl = document.body.querySelector('amp-story-player');
+           const player = new AmpStoryPlayer(window, playerEl);
+           player.load();
+	}, []);
+
+	return (
+		<div>
+		  <amp-story-player style={{ width: `${width}px`, height: `${height}px`}}>
+			<a href={url}>
+			    {title}
+			</a>
+		  </amp-story-player>
+		</div>
+	);
+}
+```
+
+#### go
+
+**Parameters**
+
+- number: the story in the player to which you want to move, relative to the current story.
+
+If the player is currently on the third story out of ten stories, `player.go(1)` will go forward one story to the fourth story and `player.go(-1)` will go backward one story to the second story. If no value is passed or if delta equals 0, current story will persist and no action will be taken.
+
+#### show
+
+**Parameters**
+
+- string: the URL of the story to show.
+
+Will change the current story being displayed by the player. If the URL is not found, an error will be thrown. This will be changed later on to automatically add the story.
+
+```
+player.show(url);
+```
+
+#### add
+
+**Parameters**
+
+- array of storyObj: contains the stories to be added.
+
+**StoryObj Type**
+
+- href string: story URL
+- title string (optional): story title, to be added to the anchor's title
+- posterImage string (optional): a URL for the story poster. Used as a placeholder while the story loads.
+
+The player will try to load the story from the cache that was specified at the player level.
```

Comment:
Please add more context as "the cache" could be different things here.

eg: "The player will rewrite the URL using the AMP Cache prefix provided in the player level attribute."